### PR TITLE
Group the config fields of the variant label

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

Just group them into a single component.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs-dev/user-guide/configuration-reference.md [hunk 1]
```diff
@@ -292,6 +289,16 @@ One of `yamlField` or `regex` is required.
 | namespace | string | The namespace where manifests will be applied. | No |
 | autoRollback | bool | Automatically reverts all deployment changes on failure. Default is `true`. | No |
 
+## KubernetesVariantLabel
+
+
```
Comment (knanao) [inline docs/content/en/docs-dev/user-guide/configuration-reference.md:294]: [nits] unnecessary new line.

Comment (nghialv) [inline docs/content/en/docs-dev/user-guide/configuration-reference.md:294]:
```suggestion
## KubernetesVariantLabel

```

Comment (nghialv) [reply docs/content/en/docs-dev/user-guide/configuration-reference.md]: Thank you. Updated.

**Comment by knanao** [comment]
Thank you very much!
Looks so good:D
/lgtm

**Comment by khanhtc1202** [comment]
Here you go
/approve

**Pull request merged**